include/event2/http.h

```diff
@@ -270,6 +270,7 @@ EVENT2_EXPORT_SYMBOL
 void evhttp_set_ext_method_cmp(struct evhttp *http,
 	evhttp_ext_method_cb *cmp);
 
+typedef void evhttp_cb(struct evhttp_request *, void *);
```

Comment:
First of all it overlaps with `evhttp_cb` in `http-internal.h` which is not a gread idea (although it requires `struct`)
Secondly it is used in other places.

So I'm going to fix this by myself, objections?

Reply:
no objection